src/components/services/CopyToClipboard.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <Generic :item="item">
+    <template #indicator>
+      <div class="status"><i class="fa-regular fa-copy fa-xl" :class="{'fa-beat': animate, 'copied-icon': animate}" @click="copy"></i></div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+
+export default {
+  name: "CopyToClipboard",
+  mixins: [service],
+  props: {
+    item: Object,
+  },
+  components: {
+    Generic,
+  },
+  data: () => ({
+    animate: false
+  }),
+  methods: {
+    copy() {
+      javascript:navigator.clipboard.writeText(this.item.clipboard)
+      this.animate = true;
+      setTimeout(
+        () => { this.animate = false },
+        1000
+      );
+    }
+  },
+};
+</script>
+
+<style scoped lang="scss">
+.status {
+  color: black;

```

Comment:
An alternative color for dark mode would be great!

Reply:
Also, maybe a litlle change of color on hover ? (has the entire card is clickable, it would confirm that you are about to activate on the action button)

Reply:
Yep, lessons learned from the proxmox service changes. I will look into bulma modifier classes because I'm not sure how to include a CSS selector that takes in count the `is-dark` class outside the component (remember, I'm not really good at frontend stuff...).